packages/terra-clinical-onset-picker/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@ ChangeLog
 
 Unreleased
 ----------
-
+q
```

Comment:
oops, looks like this accidentally got inserted

Reply:
I deleted the letter. https://github.com/cerner/terra-clinical/pull/479/commits/420780afea16476a8b8027391fc56c9a0fb36d5d#diff-d04cda2c5701de000a2f75f8a831d5b1R6